### PR TITLE
[SB] Fon study updates.

### PR DESCRIPTION
**Pull request by leonidrakitin**
Resolved DDP-7512, DDP-7518, (https://github.com/broadinstitute/ddp-study-server/pull/1635/commits/e0475c2f5f702e22c5bff0da0e8ce715dbec89c6) [DDP-7506](https://broadinstitute.atlassian.net/browse/DDP-7506)[,](https://github.com/broadinstitute/ddp-study-server/pull/1635/commits/e0475c2f5f702e22c5bff0da0e8ce715dbec89c6) [DDP-7494](https://broadinstitute.atlassian.net/browse/DDP-7494)[,](https://github.com/broadinstitute/ddp-study-server/pull/1635/commits/e0475c2f5f702e22c5bff0da0e8ce715dbec89c6) [DDP-7493](https://broadinstitute.atlassian.net/browse/DDP-7493)[,](https://github.com/broadinstitute/ddp-study-server/pull/1635/commits/e0475c2f5f702e22c5bff0da0e8ce715dbec89c6) [DDP-7501](https://broadinstitute.atlassian.net/browse/DDP-7501)[.](https://github.com/broadinstitute/ddp-study-server/pull/1635/commits/e0475c2f5f702e22c5bff0da0e8ce715dbec89c6)

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf [hunk 1]
```diff
@@ -1589,7 +1694,9 @@
         // LV Cardiac Output:
         {
           "question": {
-            include required("../../snippets/numeric-question.conf"),
+            include required("../../snippets/equation-question.conf"),
+            "expression": "(Q18_CMR_DIASTOLIC_VOLUME - Q21_CMR_SYSTOLIC_VOLUME) * Q7_CMR_HEART_RATE_BPM"
+            "maximumDecimalPlaces": 0,
```
Comment (Trimax) [inline pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf:1713]: Are you sure about this? Maximum decimal places set to 0 means that the resulting number will be rounded to the nearest integer.

File: pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf [hunk 2]
```diff
@@ -2814,28 +2992,74 @@
...
+              }
+            ],
           },
           "blockType": "QUESTION",
           "blockGuid": null,
-          "shownExpr": """user.studies["fon"].forms["CMRI"].questions["Q40_CMR_GADOLINIUM_ENHANCEMENT_PRESENT"].answers.hasOption("OPT1_POSITIVE")""",
-          "enabledExpr": null
+          "shownExpr": null,
+          "shownExpr": """user.studies["fon"].forms["CMRI"].questions["Q41_CMR_INDICATE_THE_LGE_PATTERN"].isAnswered() && user.studies["fon"].forms["CMRI"].questions["Q41_CMR_INDICATE_THE_LGE_PATTERN"].answers.hasOption("OPT6_OTHER_SPECIFY")"""
         },
```
Comment (archilsharashenidzebroad) [inline pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf:3063]: shownExpr is set twice

File: pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf [hunk 3]
```diff
@@ -2814,28 +2992,74 @@
...
+                }
+              }
+            ],
           },
           "blockType": "QUESTION",
           "blockGuid": null,
-          "shownExpr": """user.studies["fon"].forms["CMRI"].questions["Q40_CMR_GADOLINIUM_ENHANCEMENT_PRESENT"].answers.hasOption("OPT1_POSITIVE")""",
-          "enabledExpr": null
+          "shownExpr": null,
+          "shownExpr": """user.studies["fon"].forms["CMRI"].questions["Q41_CMR_INDICATE_THE_LGE_PATTERN"].isAnswered() && user.studies["fon"].forms["CMRI"].questions["Q41_CMR_INDICATE_THE_LGE_PATTERN"].answers.hasOption("OPT6_OTHER_SPECIFY")"""
```
Comment (archilsharashenidzebroad) [inline pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf:3060]: shownExpr is set twice

File: pepper-apis/studybuilder-cli/studies/fon/activities/echocardiogram.conf [hunk 4]
```diff
@@ -370,26 +385,76 @@
...
+                  ]
+                }
+              }
+            ],
           },
           "blockType": "QUESTION",
           "blockGuid": null,
           "shownExpr": null,
-          "enabledExpr": null
+          "shownExpr": """!user.studies["fon"].forms["ECHOCARDIOGRAM"].questions["Q1_ECH_CENTER_ECHOCARDIOGRAM"].isAnswered() && user.studies["fon"].forms["ECHOCARDIOGRAM"].questions["Q1_ECH_CENTER_ECHOCARDIOGRAM"].answers.hasOption("OPT12_OTHER_SPECIFY")"""
```
Comment (archilsharashenidzebroad) [inline pepper-apis/studybuilder-cli/studies/fon/activities/echocardiogram.conf:456]: shownExpr is set twice again
Is this right ?

File: pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf (hunk 3 above)
Comment (leonidrakitin) [reply pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf:3060]: thanks! fixed.

File: pepper-apis/studybuilder-cli/studies/fon/activities/echocardiogram.conf (hunk 4 above)
Comment (leonidrakitin) [reply pepper-apis/studybuilder-cli/studies/fon/activities/echocardiogram.conf:456]: thanks! fixed.

File: pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf (hunk 1 above)
Comment (leonidrakitin) [reply pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf:1713]: @Trimax So yes, Im sure, there is a formula like `(int - int) * int`

Comment (archilsharashenidzebroad) [reply pepper-apis/studybuilder-cli/studies/fon/activities/cmri.conf:1713]: echocardiogram.conf
shownExpr duplicated lines 456-457

**Review by archilsharashenidzebroad — DISMISSED** [review]

**Review by archilsharashenidzebroad — APPROVED** [review]

**Pull request merged**